patches/libcec-python313.patch

```diff
@@ -0,0 +1,22 @@
+--- a/src/libcec/SwigHelper.h
++++ b/src/libcec/SwigHelper.h
+@@ -119,7 +119,7 @@ namespace CEC
+       if (!!m_callbacks[callback])
+       {
+         /** call the callback */
+-        result = PyEval_CallObject(m_callbacks[callback], arglist);
++        result = PyObject_CallObject(m_callbacks[callback], arglist);
+
+         /** unref the argument and result */
+         if (!!arglist)
+--- a/src/libcec/libcec.i
++++ b/src/libcec/libcec.i
+@@ -141,7 +141,7 @@ namespace std {
+       if (!!lib)
+       {
+         lib->InitVideoStandalone();
+-        PyEval_InitThreads();
++        // PyEval_InitThreads();
+       }
+       return lib;
+     }
```

Comment:
Patch for Python 3.13 from fedora: https://src.fedoraproject.org/rpms/libcec/c/ef9e6d4972e0d8a9beb934a2ddf8e707bfe34507?branch=rawhide